src/com/amazon/ion/benchmark/schema/constraints/ReparsedConstraint.java

```diff
@@ -0,0 +1,5 @@
+package com.amazon.ion.benchmark.schema.constraints;
+
+public interface ReparsedConstraint {
```

Comment:
This could use a JavaDoc to explain to a reader why it exists.

Reply:
> This could use a JavaDoc to explain to a reader why it exists.

The explanation is added in the following commit.